src/manual.c

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2017 Chris Kowalczyk <ckowalczyk@suse.com>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#include "manual.h"
+
+#include "transport.h"
+#include "ticket.h"
+#include "config.h"
+#include "log.h"
+#include "request.h"
+
+
+int manual_selection(struct ticket_config *tk,
+	struct booth_site *preference, int update_term, cmd_reason_t reason)
+{
+	if (local->type != SITE)
+		return 0;
+
+	tk_log_debug("starting manual selection");
+	tk_log_debug("selection caused by %s %s",
+				state_to_string(reason),
+				reason == OR_AGAIN ? state_to_string(tk->election_reason) : "" );
```

Comment:
This is just a debug message, but still: if there are no elections on manual tickets, what can this message disclose?

Reply:
it was more to keep this consistent with election messages. I can remove it if you prefer

Reply:
Well, if there's nothing new to say... ;-) Also, perhaps it's good that the manual operations have different footprint.